pkg/launcher/paths.go

```diff
@@ -56,7 +57,14 @@ func DefaultPath(path defaultPath) string {
 	// not windows
 	switch path {
 	case RootDirectory:
-		return "/var/kolide-k2/k2device.kolide.com/"
+		const defaultRootDir = "/var/kolide-k2/k2device.kolide.com"
```

Comment:
I thought this would get read out of the config file, and this was just the default?

Reply:
with interactive we don't currently read the `launcher.flags` file, I suppose we could look for it on start up and if we don't find it, just use defaults

Reply:
That's probably reasonable. I think we implemented something like in another spot. Maybe related to the new updater.

Reply:
I guess if we go down this route, we'd need to add `root_directory ` and `config` to interactive's flags.

Reply:
I've updated it to use launcher opts + knapsack to handle figuring out root dir.